docs/_articles/en/v3.x/release-information.md

```diff
@@ -11,13 +11,14 @@ lang: en
 [Full Changelog](https://github.com/forcedotcom/sfdx-scanner/compare/v3.3.0...v3.4.0)
 
 ### Release Summary
-* Improvements to `eslint`:
-	- Execute `eslint` engine with babel parser to accommodate more modern javascript syntax
-	- Identify javascript target files that were analyzed by both `eslint` and `eslint-lwc`, and throw warning about the possibility of duplicate violations
-* Survey request banner fix
-* SFGE: Minor code fixes
-* Updates to local RetireJS Vulnerability Repository
-* PMD upgrade to 6.48.0
+
+* NEW \[SFGE\]: We now display progress information of Salesforce Graph Engine's analysis while executing `scanner:run:dfa` command.
+* NEW: If a JavaScript target file is analyzed by both `eslint` and `eslint-lwc`, we throw a warning about duplicate violations to alert you that you should modify your configuration.
+* NEW: We updated the RetireJS Vulnerability Repository.
+* NEW: We upgraded PMD to 6.48.0.
+* CHANGE: We replaced eslint's parser with `@babel/eslint-parser`
+* FIX: We removed the survey request banner's stylization
+* FIX \[SFGE\]: When Salesforce Graph Engine is unable to resolve a method call or a variable passed to a Database operation, instead of throwing an internal error, it creates a violation to let users know that they need to verify the CRUD/FLS access of the operation manually.
```

Comment:
@rmohan20 Is there a reason why "database" is capitalized? If it's not a name brand, make it lowercase: database operation. Thanks!

How about this edit? Two sentences instead of one. Really long sentences are harder for non-native English speakers, among other things. And let's be consistent and either say "you" or "them." Like above, we say "to alert you that you should ..." and generally we talk to the developer as "you." So check out this slight edit:

When Salesforce Graph Engine (SFGE) is unable to resolve a method call or a variable passed to a database operation, it no longer throws an internal error. Instead, SFGE creates a violation to let you know that you need to verify the CRUD/FLS access of the operation manually.

Reply:
By "Database", I meant this class: https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_database.htm - but since we don't check for CRUD/FLS on all methods yet, agree that we should call it "database" here.

Updating with your suggestions in a few mins...